modules/lambdas/output.tf

```diff
@@ -62,4 +62,25 @@ output "post_copy_request_to_queue_arn" {
 output "db_deploy_function_name" {
   description = "AWS Function Name for the db_deploy lambda."
   value       = aws_lambda_function.db_deploy.function_name
+}
+
+# API gateway for catalog reporting lambda
+# ------------------------------------------------------------------------------
+output "catalog_reporting_api_invoke_url" {
```

Comment:
See comments from modules/lambdas/main.tf. I am thinking we should just have a single base URL instead of three.